rts/Rendering/GL/VAO.cpp

```diff
@@ -0,0 +1,3 @@
+/* This file is part of the Spring engine (GPL v2 or later), see LICENSE.html */
+
+#include "VAO.h"
```

Comment:
Why is this file empty? GL calls should not live in headers too much.

Reply:
It nicely fit into headers.
Without the .cpp file I couldn't reference it in CMakeLists. Is there some other way?

I can move the implementation if you prefer.

Reply:
Done